.github/workflows/pr.yml

```diff
@@ -71,9 +71,10 @@ jobs:
         run: |
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33515&commitId=69e64951197bad86d5c450dd335e2df64c3578c9&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Bottom left edge.chromium.png|2309| Changed|
|Callout.Left center.chromium.png|2616| Changed|
|Callout.Top left edge.chromium.png|2307| Changed|
|Callout.No beak.chromium.png|2306| Changed|

</details>


  <details><summary><b>react-charting-VerticalBarChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-VerticalBarChart.Basic - Secondary Y Axis.chromium.png|4| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>